docs/src/modules/components/branding/Footer.js

```diff
@@ -0,0 +1,332 @@
+/* eslint-disable material-ui/no-hardcoded-labels */
+import * as React from 'react';
+import Interpolate from '@trendmicro/react-interpolate';
+import Typography from '@material-ui/core/Typography';
+import Grid from '@material-ui/core/Grid';
+import Container from '@material-ui/core/Container';
+import Divider from '@material-ui/core/Divider';
+import Link from 'docs/src/modules/components/Link';
+import Input from '@material-ui/core/Input';
+import Button from '@material-ui/core/Button';
+import IconButton from '@material-ui/core/IconButton';
+import TwitterIcon from '@material-ui/icons/Twitter';
+import GitHubIcon from '@material-ui/icons/GitHub';
+import LinkedInIcon from '@material-ui/icons/LinkedIn';
+import { useUserLanguage, useTranslate } from 'docs/src/modules/utils/i18n';
+import { experimentalStyled as styled } from '@material-ui/core/styles';
+import Logo from './Logo';
```

Comment:
I think that we should use absolute imports so we can move code around
```suggestion
import Logo from 'docs/src/modules/components/branding/Logo';
```

x the other places